operator/rpcoperator.go

```diff
@@ -129,19 +160,19 @@ func newProofSub() *ProofSub {
 	}
 }
 
-func (sub *ProofSub) Deposits() <-chan tee.DepositProof {
+func (sub ProofSub) Deposits() <-chan tee.DepositProof {
```

Comment:
why did you switch to value?

Reply:
Channel-references within the `ProofSub` don't change and are readonly, unless you view pushing into a channel as a mutable action. Imo this communicates the intent better. But it is minor, if there are any downsides to this, tell me (:

Performancewise this should not change much, since we pass pointer around anyway.